app/javascript/react/containers/ReviewContainer.js

```diff
@@ -0,0 +1,101 @@
+import React, {Component } from "react";
+import ReviewTile from '../components/ReviewTile'
+
+class ReviewContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickCounts: null
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+    handleClick(event, reviewId) {
+      let newClickCounts = this.state.clickCounts
+      if (event.target.className.includes('fa-thumbs-up')){
+        newClickCounts = this.state.clickCounts.map (clickCount => {
+          let thumbsUpClickCountThis = clickCount.thumbsUpClickCount
+          let thumbs_up_this = clickCount.thumbs_up
+          if (reviewId == clickCount.id && clickCount.thumbsUpClickCount === 0) {
+            thumbsUpClickCountThis = thumbsUpClickCountThis + 1
+            thumbs_up_this = "thumbs_up"
+          }
+          return {
+            id: clickCount.id,
+            thumbsUpClickCount: thumbsUpClickCountThis,
+            thumbsDownClickCount: 0,
+            thumbs_up: thumbs_up_this,
+            thumbs_down: 0
+          }
+        })
+      } else if (event.target.className.includes('fa-thumbs-down')){
+        newClickCounts = this.state.clickCounts.map (clickCount => {
+          let thumbsDownClickCountThis = clickCount.thumbsDownClickCount
+          let thumbs_down_this = clickCount.thumbs_down
+          if (reviewId == clickCount.id && clickCount.thumbsDownClickCount === 0) {
+            thumbsDownClickCountThis = thumbsDownClickCountThis + 1
+            thumbs_down_this = "thumbs_down"
+          }
+          return {
+            id: clickCount.id,
+            thumbsUpClickCount: 0,
+            thumbsDownClickCount: thumbsDownClickCountThis,
+            thumbs_up: 0,
+            thumbs_down: thumbs_down_this
+          }
+        })
+      }
+      this.setState({clickCounts: newClickCounts})
+    }
+
+  render() {
+    if ((this.state.clickCounts == null) && (this.props.reviews.length > 0)) {
+      let clickCountsInitial = this.props.reviews.map (review => {
+        return {
+          id: review.id,
+          thumbsUpClickCount: 0, //0 for now; eventually need to read from db
```

Comment:
is this currently working? it looks like it shouldn't be. Be sure to also remove the comments